apps/rellotge/metadata.json

```diff
@@ -0,0 +1,15 @@
+{ "id": "rellotge",
+  "name": "Rellotge en catala",
+  "shortName":"Rellotge",
+  "version":"0.4",
+  "description": "A catalan clock with traditional naming of hous",
+  "icon": "icona.js",
```

Comment:
Two fixes:

* catalan should be written "Catalan". Names of languages are written always in upper case in English. 
* hous should be "hours"